src/engine/client/cl_main.cpp

```diff
@@ -1897,7 +1897,7 @@ CL_ServerLinksResponsePacket
 */
 void CL_ServerLinksResponsePacket( msg_t *msg )
 {
-	int      port;
+	unsigned short      port;
```

Comment:
Can we use uint16_t here and elsewhere ?

Reply:
in documentation https://en.cppreference.com/w/cpp/language/types and https://en.cppreference.com/w/c/types/integer . it's always equal on all platforms so yes, you can replace all unsigned short of the codebase with uint16_t. it's a small fix, so i try to change only the needed code.

Reply:
No, I don't mean the codebase. Just where you touch it for this change. I find uint16_t more readable and explicit.

Reply:
all changes made